app/controllers/brapi/v1/germplasm_controller.rb

```diff
@@ -2,68 +2,64 @@ class Brapi::V1::GermplasmController < Brapi::BaseController
 
   attr_accessor :request_params, :user
 
-  before_filter :authenticate_api_key!
+  # Authentication via ORCID is currently unsupported by BrAPI.
```

Comment:
To clarify, BIP API uses internal authentication. ORCID is used only for the UI.